wasmVirtualMachine.js

```diff
@@ -372,11 +375,56 @@ function build_module(byte_code) {
                 i++;
                 decode = Leb.decodeUint32(byte_code, i);
                 i = decode.nextIndex;
-                size = decode.value;
+                expected_end = i + decode.value;
+
+                // get array of table descriptions
+                decode = Leb.decodeUint32(byte_code, i);
+                i = decode.nextIndex;
+                num_els = decode.value;
+
+                // get table description
+                for (let j = 0; j < num_els; j++) {
+                    // table element type
+                    type = byte_code[i];
+                    i++;
+
+                    // // sanity check
+                    // if (type != -0x10) {
```

Comment:
Add the sanity check back in, but the type should be 0x70
https://webassembly.github.io/spec/binary/types.html#binary-tabletype